### ISSUE TITLE
jellyfin armada38x not working

### ISSUE THREAD

**Issue by nerojdn** [publicarray/spksrc#27]
### Setup
_Package Name:_ jellyfin
_Package Version:_ 10.6.4

_NAS Model:_ DS116
_NAS Architecture:_ armada38x
_DSM version:_ 6.2.2

### Expected behavior
Jellyfin package launchs to execution

### Actual behavior
jellyfin doesn't launch

### Steps to reproduce
_1._ Install docker, etc... 
_2._ make spk-jellyfin-armada38x-6.2.2
_3._ install package generated
_4._ jellyfin is not executing, click over exec button and try again and again....

#### Package log
_`/usr/local/jellyfin/var/jellyfin.log`_
```
Mon Feb  8 21:20:30 CET 2021
Starting jellyfin command /volume1/@appstore/jellyfin/jellyfin --noautorunwebapp --package-name synology -d /volume1/@appstore/jellyfin/var/data -C /volume1/@appstore/jellyfin/var/cache -c /volume1/@appstore/jellyfin/var/config -l /volume1/@appstore/jellyfin/var/log -w /volume1/@appstore/jellyfin/web 
/volume1/@appstore/jellyfin/jellyfin: /lib/libstdc++.so.6: no version information available (required by /volume1/@appstore/jellyfin/jellyfin)
/volume1/@appstore/jellyfin/jellyfin: /lib/libstdc++.so.6: no version information available (required by /volume1/@appstore/jellyfin/jellyfin)
/volume1/@appstore/jellyfin/jellyfin: /lib/libstdc++.so.6: no version information available (required by /volume1/@appstore/jellyfin/jellyfin)
/volume1/@appstore/jellyfin/jellyfin: /lib/libstdc++.so.6: no version information available (required by /volume1/@appstore/jellyfin/jellyfin)
/volume1/@appstore/jellyfin/jellyfin: /lib/libstdc++.so.6: no version information available (required by /volume1/@appstore/jellyfin/jellyfin)
The application to execute does not exist: '/volume1/@appstore/jellyfin/jellyfin.dll'.
```

### Aditional notes
publicarray jellyfin-10.6.2 release was working like a charm, except in my webos2 LG (scrolling is a disaster, waiting for 10.7 releases) 

If I can try another config, please tell me. Same if you need more info.

Thanks!!!

**Comment by publicarray** [comment]
Where did you download 10.6.4 from? was it from here https://seby.io/download/synology-7.0? If yes then you need DSM7 for that build. The compiler and stdlib is newer as shown by the error message.
I can push a build for the 6.x DSM if you like.

**Comment by publicarray** [comment]
While refactoring the makefiles a while ago I missed updating a variable. Can you give it another try? 
Note I used a force bush to also rebased with the master branch.

**Comment by nerojdn** [comment]
> Where did you download 10.6.4 from? was it from here https://seby.io/download/synology-7.0? If yes then you need DSM7 for that build. The compiler and stdlib is newer as shown by the error message.
> I can push a build for the 6.x DSM if you like.
> 
> Edit: never mind should have read the whole issue. I'll investigate.

**Comment by publicarray** [comment]
:grey_question:

**Comment by publicarray** [comment]
No it's all good :) I've just been working on DSM7 for the last few weeks

**Comment by publicarray** [comment]
It's building now: https://github.com/publicarray/spksrc/actions/runs/549572895

**Comment by nerojdn** [comment]
Sorry, I'm novice in github and I'm surprised: remote building!! (there's another  world beyond Microsoft ahaha)
Thanks a lot man! Tomorrow (in a few hours) I'll try again and tell you

**Comment by publicarray** [comment]
No worries, Did you know, [Microsoft bought GitHub in 2018](https://github.blog/2018-10-26-github-and-microsoft/)? Can't escape the orbit so easily :laughing:

**Comment by nerojdn** [comment]
Definitely ahahah

**Comment by nerojdn** [comment]
I'm just probe it! Perfect! Like your 10.6.2.
However, my webos 2 tv still has the same scrolling problem. We will have to wait for 10.7 or a new android TV (never LG again)
Thank you so much, your work is incredible!

**Comment by publicarray** [comment]
Thank you! And thank you for your report!

Good luck with your TV, Is there already a fix for it in 10.7? If not can you create a bug report for it?
